e2e/fixtures/README.md

```diff
@@ -0,0 +1,32 @@
+# End-to-end tests
+
+This directory contains the fixture data used to bootstrap the individual e2e tests. Each sub-directory contains
+two things:
+
+1. A `state.json` file that represents a the saved state for the extension (see _Generating fixture data_ below)
+2. A `db.js` file that exports a config for Ganache
+
+The `db/` sub-sub-directory and its contents, should it exist, contains Ganache's state and a path to it is configured
+in the `db.js` file.
+
+## Generating fixture data
+
+Fixture data can be generated by loading the unpacked extension, inspecting the background context,
+dumping `chrome.storage.local`, and using [`copy`][1] to copy it to the clipboard, and manually pasting the contents into
+a file on disk.
+
+```js
+chrome.storage.local.get(null, (x) => console.log(x))
+// Store the value as a global (shown below)
+copy(temp1)
+```
+
+Store the value as a global variable:
```

Comment:
It was a little unclear to me which order this was supposed to be done in. Is this correct?

1. Using an unpacked extension, get the extension into whatever state you want to save
2. Inspect the background page
3. Dump `chrome.storage.local` to the console using `chrome.storage.local.get(null, (x) => console.log(x))`. It should display an object of the form `{data: {...}, meta: {...}}`
4. Right-click on `data:` and select `Store the value as a global variable`. It should store it as `temp1`
5. Copy the variable to the clipboard using `copy(temp1)

The instructions as written make it sound like steps 4 and 5 are swapped.

Reply:
Upon re-reading it, it does seem kinda obvious what was meant with the `(shown below)` part - not sure how I missed that. Maybe I was just mislead by that step being missing from the paragraph above.